NEWS

```diff
@@ -1,3 +1,27 @@
+30-Nov-2018 IGNITION v0.29.0
+
+  Features:
+
+    - Add support for ?versionId on s3:// URLs
+
+  Changes:
+
+    - Mark the 2.3.0 config spec as stable
+    - No longer accept configs with version 2.3.0-experimental
+    - Create new 2.4.0-experimental config spec from 2.3.0
+
+  Bug Fixes:
+
+    - Fall back to /dev/urandom instead of blocking indefinitely when more entropy
+      is needed for TLS
+    - Fix race with umount when running blackbox tests
```

Comment:
Can we start moving test changes to a separate section?  They're not user-facing.